test_data/generated/show/hello.json

```diff
@@ -50,88 +54,96 @@
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "x86_64-linux",
+        "system": "aarch64-linux",
         "version": "2.12"
       },
       {
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "aarch64-darwin",
+        "system": "x86_64-linux",
         "version": "2.12"
       },
       {
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "x86_64-darwin",
+        "system": "aarch64-darwin",
         "version": "2.12"
       },
       {
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "aarch64-linux",
+        "system": "x86_64-darwin",
         "version": "2.12"
       },
       {
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "x86_64-linux",
+        "system": "aarch64-linux",
         "version": "2.10"
       },
       {
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "aarch64-darwin",
+        "system": "x86_64-linux",
         "version": "2.10"
       },
       {
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "x86_64-darwin",
+        "system": "aarch64-darwin",
         "version": "2.10"
       },
       {
         "description": "A program that produces a familiar, friendly greeting",
         "input": "nixpkgs",
         "license": "GPL-3.0-or-later",
+        "pkgPath": "nixpkgs/hello",
         "pname": "hello",
         "relPath": [
           "hello"
         ],
-        "system": "aarch64-linux",
```

Comment:
blocking: can you add a test (or explain why you shouldn't)?

Reply:
I pushed a couple of tests of the output

Reply:
Thanks! Nonblocking but I think it might be worth generating and loading a mock, because I think the part more likely to break is something like we're confused about what the API returns (e.g. is nixpkgs in pkg_path?)

Reply:
Although maybe there aren't any packages we can test against? 🤔